pootle/apps/pootle_store/forms.py

```diff
@@ -279,16 +283,42 @@ def clean_target_f(self):
 
             return value
 
-        def clean_state(self):
+        def clean_similarity(self):
```

Comment:
They were not just moved. 
`clean_state()` functionality was moved to `clean()` because `clean_A()` shouldn't depend on `clean_B()`.